sru/sru_functional.py

```diff
@@ -346,13 +365,20 @@ def forward(self, input, c0=None, mask_pad=None):
         if self.layer_norm:
             input = self.layer_norm(input)
 
+        # project if needed
+        if self.input_to_hidden is not None:
+            input = self.input_to_hidden(input)
+
```

Comment:
residual should have dimension = output_size as well. so do this before layer norm is applied:
```
if self.input_to_hidden is not None:
    input = self.input_to_hidden(input)

residual = input
if self.layer_norm:
    input = self.layer_norm(input)
```

Reply:
avoids layer norm on the skip path.